ConnectionManager.class.nut

```diff
@@ -274,28 +289,46 @@ class ConnectionManager {
         }
 
         // Run the global onConnected Handler if it exists
-        if (_onConnect != null) {
```

Comment:
Personally I'd prefer the way it looked before. It's more readable, even though it costs a couple of additional lines of code. But we we don't seem to be charged for them )

Anyway, it's a minor cosmetic comment (as well as most of others or mine), that I don't insist on fixing.